### PR TITLE
SHOW statements docs (part II)

### PR DESCRIPTION
**Pull request by randomJoe211**
The rest of the `SHOW` docs.

### CODE REVIEW

**Review by whitewum — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/10.show-roles.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# SHOW ROLES Syntax
+
+The `SHOW ROLES` statement shows the roles that are assigned to a user account.
+
+The return message differs according to the role of the user who is running this statement:
+
+* If the user is a `GOD` or `ADMIN` and is granted access to the specified graph space, Nebula Graph shows all roles in this graph space.
+* If the user is a `DBA`, `USER` or `GUEST` and is granted access to the specified graph space, Nebula Graph shows the user's own role in this graph space.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/10.show-roles.md:8]: , or `GUEST`

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/11.show-snapshots.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# SHOW SNAPSHOTS
+
+The `SHOW SNAPSHOTS` statement shows all the snapshots.
+
+For how to create a snapshot and backup data, see CREATE SNAPSHOT (TODO).
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/11.show-snapshots.md:5]: back up

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/16.show-users.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# SHOW USERS
+
+The `SHOW USERS` statement shows the user information. `SHOW USERS` output has these columns: account names.
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/16.show-users.md:3]: columns? plural?

File: docs-2.0/3.ngql-guide/7.general-query-statements/6.show/9.show-parts.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+The `SHOW PARTS` statement shows the information of a specified partition or all partitions in a graph space.
+
+## Syntax
+
+```ngql
+SHOW PARTS [<part_id>]
+```
+
+## Examples
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/7.general-query-statements/6.show/9.show-parts.md:11]: put "SHOW PARTS" before "SHOW PARTS 1"? As a user, when I read "SHOW PARTS 1" example, I would want to know how to get the partition number.

**Review by oldLady344 — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**